components/card/src/card.tsx

```diff
@@ -31,7 +31,7 @@ CardBody.displayName = "CardBody";
  * The footer container of a card. Usually composed as a direct child of a `Card` component.
  */
 export const CardFooter = forwardRef<HTMLDivElement, CardProps>(({ className, children, ...rest }, ref) => (
-	<div ref={ref} className={cx("border-t border-gray-300 px-6 py-3", className)} {...rest}>
+	<div ref={ref} className={cx("border-t border-card-muted px-6 py-3", className)} {...rest}>
```

Comment:
We want to bump this one up a bit otherwise we'll have this regression with the current high contrast clamping. 
<img width="1940" alt="image" src="https://github.com/ngrok-oss/mantle/assets/1369864/eb9dd89a-422f-4e6a-b3b3-156f460a5aef">

Reply:
cool. in HC modes, I think maybe  we can just make `muted` not "exist", and just make all the borders the same. value.